api/internal/humiographql/user.go

```diff
@@ -0,0 +1,7 @@
+package humiographql
+
+import graphql "github.com/cli/shurcooL-graphql"
+
+type User struct {
+	ID graphql.String `graphql:"id"`
```

Comment:
Need this in aggregate alert. Forgot to revert it when cherry-picking. Including this now is fine.